implementations/rust/ockam/ockam_command/tests/bats/unit.bats

```diff
@@ -163,16 +163,14 @@ teardown() {
   # Create with random name
   run "$OCKAM" identity create
   assert_success
-  assert_output --partial "Identity created"
 
   # Create a named identity and delete it
   i=$(random_str)
   run "$OCKAM" identity create "${i}"
   assert_success
-  assert_output --partial "Identity created"
```

Comment:
These were removed because of the way `run` works within bats. 

The command and output interprets it as a tty, and then prints the machine output.

Reply:
Something that we should discuss is that we might need to change the output to show the identity name instead of the identifier. That would make things easier. Right now we can't do anything after creating an unnamed identity. If we do this change we could:

```bash
name = $(ockam identity create)
identifier = $(ockam identity show $name)
```